src/planscape/forsys/write_debug_data_to_disk.R

```diff
@@ -0,0 +1,62 @@
+suppressMessages({
+    library(dplyr)
+    library(ggplot2)
+    library(ggnewscale)
+    library(sf)
+})
+
+write_debug_data_to_disk <- function(output_dir,
+                                     forsys_input_data,
+                                     run_outputs,
+                                     conditions,
+                                     wp_colname,
+                                     stand_id_field,
+                                     proj_id_field) {
+    # Writes the input to a shape file.
+    st_write(
+      obj = forsys_input_data,
+      file.path(output_dir, "forsys_input_data.shp"))
+
+    # Graphs conditions and weighted priorities.
+    for (p in conditions) {
+      ggplot(data = forsys_input_data) + 
+        geom_sf(mapping = aes_string(fill = p), color = NA) +
+        scale_fill_viridis_c(begin = 0, end = 1, option = "turbo") +
+        guides(fill = guide_colorbar(title = p))
+      ggsave(file.path(output_dir, paste(p, ".pdf")))
+    }
+    ggplot(data = forsys_input_data) + 
+      geom_sf(mapping = aes_string(fill = wp_colname), color = NA) +
+      scale_fill_viridis_c(begin = 0, end = 1, option = "turbo") +
+      guides(fill = guide_colorbar(title = wp_colname))
+    ggsave(file.path(output_dir, paste(wp_colname, '.pdf')))
+
+    # Gets projects.
+    stand_output_ids <- run_outputs$stand_output %>%
+      select({{stand_id_field}}, {{proj_id_field}})
+    stand_output_ids[stand_id_field] <- 
+      lapply(stand_output_ids[stand_id_field], as.integer)
```

Comment:
Why does the `stand_id` have to be cast as integer again? This already happened [here](https://github.com/OurPlanscape/Planscape/pull/635/files#diff-56edc0d8b58521fc4e1a1dd9a1acfcca16e51c2784eb5c35494c8592fd3f4763R105)(line 105)

Reply:
the mutate syntax wasn't working for me after removing tidyverse - maybe I should remove it there, too
a TODO once I go through all the comments

Reply:
update: I've removed the type case for stand_id_field.

I didn't bother replacing the "mutate({{" calls with mutate_at but might someday in the future.